samples/MQL4/MovingAverages.mqh

```diff
@@ -0,0 +1,240 @@
+//+------------------------------------------------------------------+
+//|                                               MovingAverages.mqh |
+//|                   Copyright 2009-2013, MetaQuotes Software Corp. |

```

Comment:
Yes. Samples need to be explicitly available for distribution. Specifically, released under a permissive license like MIT, ISC, Apache-2, etc. The full [whitelist is here](https://github.com/github/linguist/blob/master/vendor/licenses/config.yml).